contracts/eosio.msig/src/eosio.msig.cpp

```diff
@@ -113,6 +132,28 @@ void multisig::unapprove( name proposer, name proposal_name, permission_level le
             a.provided_approvals.erase( itr );
          });
    }
+
+   proposals proptable( get_self(), proposer.value );
+   auto& prop = proptable.get( proposal_name.value, "proposal not found" );
+
+   if ( prop.earliest_exec_time.has_value() ) {
+      if ( prop.earliest_exec_time.value().has_value() ) {
+         auto table_op = [](auto&&, auto&&){};
+         if ( approvals_satisfy_trx_authorization(proposer, proposal_name, prop.packed_transaction, table_op) ) {
+            return;
+         } else {
```

Comment:
Rather than the `return` in the line above, I would like to see this else branch instead guarded by `if( !approvals_satisfy_trx_authorization(proposer, proposal_name, prop.packed_transaction) ) {`.

Reply:
Previously linked: https://github.com/EOSIO/eosio.contracts/blob/deprecate-deferred3/contracts/eosio.msig/src/eosio.msig.cpp#L145-L155